relatio/supported_models.py

```diff
@@ -0,0 +1,22 @@
+language_models = {
```

Comment:
This can be a constant dictionary in the embeddings module.

Reply:
I would rather not (in the future, the language information will be useful for other modules, too).

Reply:
ok. I will still make it a clear constant.